src/views/List/List.jsx

```diff
@@ -164,7 +192,12 @@ export default function List({ token }) {
                     id={name}
                     name={listItem.id}
                   />{' '}
-                  <label htmlFor={name}>{name}</label>
+                  <label htmlFor={name}>
+                    {name}{' '}
+                    <small className="badge">
+                      <strong>{badge}</strong>
```

Comment:
nice semantic HTML here and is dynamically conveying the categories! Super dry clean code, you can set the badge in one place and it will display on all items!

My only suggestion is a user who is using the app for the first time may now know what "Kind-of-Soon" next to each item name means 

so as we're going into the next 2 weeks where you get to flush out the functionality and user experience think about ways you could convey to the user what this tag represents; one example is a statement like "We estimate that you will need this item again: ___soon___" 

there's like 1,001 ways to give users that amount of additional info so they understand what data you're giving them so will be excited to see how the team builds on your work this week!